Implement the review feedback from the second round (#53)
UBC-MDS/teaching_responsibility: manuscript/teaching-reproducibility.Rmd

(e.g., "You are in 'detached HEAD' state. You can look around, make experimental changes and commit them, and you can discard any commits you make in this state without impacting any branches by performing another checkout").
Sometimes these situations are so difficult to get out of, that even professional
data scientists and data science educators
recommend the practice of "burning it all down",

> i.e. deleting the local copy of the repository

Reviewer: ```suggestion
which is deleting the local copy of the repository
```

Yet another example of a popular reproducibility tool that is challenging to learn is
Docker.
Docker is a containerization software that extends computational environment management beyond
the programming language and package dependencies of your analysis workflow,
and instead creates a versioned copy of your entire computational environment
including any additional software and operating system dependencies that your analysis may depend on.
This means that you can share an exact copy of your computational environment

> with your collaborators, even if you work on different operating systems,

Reviewer: ```suggestion
with your collaborators, even if you work on different operating systems.
```

> which is the most robust strategy for ensuring
> that your analysis will be reproducible
> and thus highly beneficial for students to master.

Reviewer: ```suggestion
Recording and sharing computational environments 
via containerization is currently viewed as the most 
robust, yet also light-weight strategy, for ensuring
an analysis will be reproducible.
It is also widely used in industry for deploying software.
Thus for several reasons, it is a highly beneficial technology for students to learn.
```

Unfortunately,

> Docker is an especially challenging tool to

Reviewer: ```suggestion
Docker, as well as other implementations of containerization, 
is an especially challenging tool to
```
